bazel/cc_configure.bzl

```diff
@@ -1,56 +1,48 @@
-# This file was imported from https://github.com/bazelbuild/bazel at 3d00b2a. We apply a number of
-# local modifications to deal with known issues in Bazel 0.10.0.
-#
-# * https://github.com/bazelbuild/bazel/issues/2840
-# * (and potentially) https://github.com/bazelbuild/bazel/issues/2805
-#
-# See cc_configure.bzl.diff for the changes made in this fork.
+load("@bazel_tools//tools/cpp:cc_configure.bzl", _upstream_cc_autoconf_impl = "cc_autoconf_impl")
+load("@bazel_tools//tools/cpp:lib_cc_configure.bzl", "get_cpu_value")
+load("@bazel_tools//tools/cpp:unix_cc_configure.bzl", "find_cc")
 
-# Copyright 2016 The Bazel Authors. All rights reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-"""Rules for configuring the C++ toolchain (experimental)."""
+def _quiet_fake_which(program):
```

Comment:
Can you add comments here and below providing details on why we're faking?

Reply:
Done.